docs/docs-content/clusters/edge/local-ui/cluster-management/link-hosts.md

```diff
@@ -0,0 +1,157 @@
+---
+sidebar_label: "Link Hosts"
+title: "Link Hosts"
+description: "Instructions for linking hosts to prepare for multi-node cluster creation."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["edge"]
+---
+
+To create a multi-node cluster with hosts provisioned in `airgap` installation mode, the hosts must first be able
+identify and securely communicate with each other. By default, hosts that are provisioned in `airgap` installation mode
+are not aware of each other even if they are on the same network, and they do not have the credentials to communicate
+with each other securely.
+
+Host linking provides the hosts with the necessary network and security infrastructure to form a cluster together. In a
+group of linked hosts, hosts can broadcast information to all its peers. Every group has a leader node, which provides
+tokens with its IP and a One-Time Password (OTP) credentials encrypted that you can use to link other nodes.
+
+Linked hosts will sync images for the Palette agent, provider images, cluster definitions, as well as host status with
```

Comment:
We can maybe mention - The content uploaded on any leader will be automatically synced to all the linked hosts. This content will include the core images for agent and provider. The cluster config that is upload during cluster creation will only be synced to hosts that are part of the cluster or machine-pools configuration.